README.md

```diff
@@ -23,19 +23,28 @@ Typescript client libraries for interacting with a Nillion cluster.
 - HTTP headers are needed in order for the browser to load the wasm bundle:
    - `Cross-Origin-Embedder-Policy: require-corp`
    - `Cross-Origin-Opener-Policy: same-origin`
-- The nilchain spawned with `nillion-devnet` does not support CORS. The recommended workaround is proxy requests to nilchain for local development. 
+- The nilchain spawned with [nillion-devnet](https://docs.nillion.com/nillion-devnet) does not support CORS. The recommended workaround is proxy requests to nilchain for local development. 
 
 ## Quick start
 
 Complete examples are available at [examples/react](https://github.com/NillionNetwork/client-ts/tree/main/examples/react) or [examples/nextjs](https://github.com/NillionNetwork/client-ts/tree/main/examples/nextjs).
 
-1. Add nillion dependencies to a basic React + webpack project.
+
+1. Ensure you are running `nillion-devnet` in another terminal window.
```

Comment:
Understand it is a pre-requisite, but was useful for me to remember when spinning up my project.